### PR TITLE
Readme: add new commands

### PR DESCRIPTION
**Pull request by rhombl4**
Update a readme file with new commands that were added in previous releases.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,7 +124,8 @@ Command     | Aliases         | Subcommands
 `method`    | `m`             | `instance`
 `next`      | `n`             |
 `pry`       |                 |
-`quit`      | `q`             |
+`quit`      | `q`             | `!`
```
Comment (deivid-rodriguez) [inline README.md]: I guess you're adding `!` as a subcommand because `quit !` and `q !` (command followed by a space and then `!`) works?

Comment (rhombl4) [reply README.md]: Yes, it works. Also, there is subcommand `unconditionally`(with the same behavior as `!`). I can add this too.
And the same subcommands for a command `continue`.

Comment (rhombl4) [reply README.md]: It is an option for command, not a sub-command. Canceled this change.

**Comment by rhombl4** [comment]
Now, these commands are looking as duplicate. But they should be present in this list, otherwise, users will not be able to learn about these helpful commands.

**Pull request merged**